ts/Dashboards/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "../tsconfig.json",
```

Comment:
We have the dashboard config in `ts/masters-dashboards`.

Why do we need this `tsconfig.json`?

Reply:
Maybe you can replace it with a `tslint.json`? https://palantir.github.io/tslint/usage/configuration/

Reply:
Hm, but the tsconfig will be added here -> https://github.com/highcharts/highcharts/pull/19355
So does it ultimately matter not to add it?